Pod/Classes/GDGColumn.m

```diff
@@ -58,4 +57,16 @@ - (NSString *)fullName
 	return [self.source.alias stringByAppendingFormat:@".%@", _name];
 }
 
+- (GDGColumn *)copyWithZone:(nullable NSZone *)zone
+{
+	GDGColumn *copy = [(GDGColumn *) [[self class] alloc] init];
```

Comment:
It should be `allocWithZone:zone`
&&
You are being more verbose that if you just write `[[GDGColumn alloc] init]`